src/templates/customer/edit_account/template.html

```diff
@@ -173,7 +173,7 @@ <h1>Edit Your [@config:company_name@] Account Details</h1>
 				}
 
 				if(count <= 0) {
-					msg.innerHTML = '<span class="text-danger">Invalid Post Code.</span>';
+					msg.innerHTML = '<span class="text-danger">Invalid Postal Code.</span>';
```

Comment:
Why not do the same 
`[%if [@config:defaultcountry@] eq 'US'%]Zip [%else%]Postal [%/if%]`
here?